### ISSUE TITLE
Extension doesn't save images in Windows

### ISSUE THREAD

**Issue by petrdvor** [mushanshitiancai/vscode-paste-image#99]
When the paste action is triggered either by shortcut or from the command pallete, the 'asssets' folder or any nested folder is created but the image is not created/saved. The PowerShell is in the PATH variable.
VS Code version: 1.58.2, Win10

**Comment by ksprashu** [comment]
I am able to reproduce this. 
Do you see the following message as well when you try this? 
![image](https://user-images.githubusercontent.com/476729/130324345-358ff1f6-9c3d-4458-ae49-d354afda1a10.png)

I am using Window + WSL.

**Comment by petrdvor** [comment]
No, I don't get any message. Only the folder is created, but the picture is not there nor shown as a link in the md text.

**Comment by kelvin-cheng** [comment]
I ran into a similar issue on Windows 10. Root cause appears to be anti-virus software quarantining a Powershell script for the extension: `%userprofile%\.vscode\extensions\mushan.vscode-paste-image-1.0.4\res\pc.ps1`.

**Comment by ksprashu** [comment]
Interesting... I don't have a `mushan*` folder in my `%userprofile%\.vscode\extensions` path.

**Comment by samxue** [comment]
I think i encount the same problem with following procedures.
===
step1:  paste one image to clickboard
step2: as the extension 'paste image' described, use the'Ctrl+ALT+V' to insert the image in the markdown file.
step3: it prompt the error 'failed make folder message=spawn EPERM'

I guess it is related with the feature of https://code.visualstudio.com/docs/editor/workspace-trust
I try to add the  config 'security.workspace.trust.emptyWindow=fasle' in the setting.  but it still doesn't work.

**Comment by kovalroma** [comment]
Hello, the same problem is here

![2021-11-25_09h22_24](https://user-images.githubusercontent.com/20167720/143405667-f9cda2dd-1629-450f-a1d3-08ac081aece4.png)

**Comment by kovalroma** [comment]
![2021-11-25_09h26_24](https://user-images.githubusercontent.com/20167720/143405816-c13007d2-5f6e-4402-b1fe-c926256608ff.png)

**Comment by vezaynk** [comment]
@kovalroma @petrdvor @samxue

This repository has not seen any activity in over 3 years. There is an actively maintained fork here: https://github.com/telesoho/vscode-markdown-paste-image

Your use-case works in the fork. I have personally implemented WSL support yesterday.

@kelvin-cheng Other people did some work for win10. I would suggest you check for the fork as well. No promises.
